app/helpers/fhir_helper.rb

```diff
@@ -262,9 +264,10 @@ def language_coding(language)
     PatientHelper.languages(language&.downcase) ? FHIR::Coding.new(**PatientHelper.languages(language&.downcase)) : nil
   end
 
-  # Helper to understand an extension for isolation status
-  def from_isolation_extension(patient)
-    patient&.extension&.select { |e| e.url.include?('isolation') }&.first&.valueBoolean == true
+  # Helper to understand a boolean extension which defaults to false
+  # NOTE: This function should only be used on fields for which nil/null is equivalent to false
+  def from_default_false_bool_extension(patient, extension_id)
+    patient&.extension&.find { |e| e.url.include?(extension_id) }&.valueBoolean == true
```

Comment:
Sorry to be nitpicky/make you change this again, but I think I'd actually prefer if this were
```suggestion
  def from_bool_extension(patient, extension_id)
    patient&.extension&.find { |e| e.url.include?(extension_id) }&.valueBoolean
```

and then in the places where we want false default, we can just call like:
```ruby
continuous_exposure: from_bool_extension(patient, 'continuous-exposure') || false
```

Reply:
No worries, updated in https://github.com/SaraAlert/SaraAlert/pull/709/commits/8f204fc4524edee712934e61267cc6ec48a9cc5d.